### ISSUE TITLE
(📚) The readme on pypi is quite minimal

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#12456]
**Documentation**

![image](https://user-images.githubusercontent.com/65446343/160223316-4fe88ac0-00e1-4ff4-8b98-c910ca952468.png)

**Comment by JelleZijlstra** [comment]
Yes, this isn't great. It's hardcoded in setup.py. Maybe we should reuse README.md instead, or at least link to the docs website.

**Comment by AlexWaygood** [comment]
Pytype does a really good job with their PyPI readme: https://pypi.org/project/pytype/
